src/Parse/Lexer.x

```diff
@@ -84,7 +84,9 @@ tokens :-
   \>\=                                  { mapToken (\_ -> TokenRightChevronEq) }
   \<\=                                  { mapToken (\_ -> TokenLeftChevronEq) }
   \!                                    { mapToken (\_ -> TokenExclamationMark) }
-  \"($printable # \")*\"                { mapToken (\s -> TokenStr (sanitizeStr s)) }
+  \"($printable # \")+\"                { mapToken (\s -> TokenStr (sanitizeStr s)) }
+  '($printable # ')+\'                  { mapToken (\s -> TokenStr (sanitizeStr s)) }
+  `($printable # `)+\`                  { mapToken (\s -> TokenStr (sanitizeStr s)) }
```

Comment:
Have you tested these ? I find it weird that we have to escape the ones on the right but not on the left ?

Reply:
I have not tested them explicitly, but I have been writing strings in all variety of quotes and I haven't seen any issues yet. I can add a test case though.